src/main/java/thePackmaster/cards/graveyardpack/FuneraryJewels.java

```diff
@@ -0,0 +1,49 @@
+package thePackmaster.cards.graveyardpack;
+
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.DexterityPower;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class FuneraryJewels
```

Comment:
This card seems kind of bad. Could probably stand to increase block a little or do something else with it

Reply:
I guess I could change it to the slightly more busted version that costs 1 and refunds you 1 energy.